testing/scripts/seldon_e2e_utils.py

```diff
@@ -209,7 +227,7 @@ def initial_grpc_request(
             rows=rows,
             data=data,
         )
-    except:
+    except Exception:
```

Comment:
No need to add `Exception` if you don't use its value to print sth.

Reply:
The `flake8` linter complains if it's not there. It's actually part of [PEP8](https://www.python.org/dev/peps/pep-0008).

We are not using `flake8` on the project but I do locally, so I try to complain with it whenever I can.

Reply:
Which point are you referring to? Do you meant `When catching exceptions, mention specific exceptions whenever possible instead of using a bare except: clause:`? 

My point is that bare `except` and `except Exception` are exactly the same as [Exception](https://docs.python.org/3/library/exceptions.html#Exception) is a base class for almost all other exceptions. I guess `flake8` wants you to put there more meaningful one like `ImportError` or `ValueError`.

Reply:
Ah, yes, you're right. Didn't read thoroughly enough. 
Let's keep `except Exception` then.